<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="LAPTOP-G76E8532" failures="1" tests="3" name="Test" time="7.798" errors="0" timestamp="2024-12-21T20:01:41 IST">
  <testcase classname="day46.OrangeHRM" name="testLogo" time="0.031"/>
  <testcase classname="day46.OrangeHRM" name="testAppUrl" time="0.004">
    <failure type="java.lang.AssertionError" message="expected [https://opensource-demo.orangehrmlive.com/] but found [https://opensource-demo.orangehrmlive.com/web/index.php/auth/login]">
      <![CDATA[java.lang.AssertionError: expected [https://opensource-demo.orangehrmlive.com/] but found [https://opensource-demo.orangehrmlive.com/web/index.php/auth/login]
at day46.OrangeHRM.testAppUrl(OrangeHRM.java:44)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- testAppUrl -->
  <testcase classname="day46.OrangeHRM" name="testHomePageTitle" time="0.0">
    <skipped/>
  </testcase> <!-- testHomePageTitle -->
</testsuite> <!-- Test -->
